operations/cancellationpolicies_new rateGroups.md

```diff
@@ -0,0 +1,142 @@
+# Cancellation policies
+
+## Get cancellation policy configurations
+
+> ### Restricted!
+> This operation is currently in beta-test and as such it is subject to change.
+
+Returns cancellation policy configurations including exceptions for rate groups and their rates, filtered by services, and other filters.
+
+Note this operation uses [Pagination](../guidelines/pagination.md) and supports [Portfolio Access Tokens](../guidelines/multi-property.md).
+
+### Request
+
+`[PlatformAddress]/api/connector/v2/cancellationPolicies/getCurrentConfiguration`
+
+```javascript
+{
+    "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+    "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+    "Client": "Sample Client 1.0.0",
+    "EnterpriseIds": [
+        "3fa85f64-5717-4562-b3fc-2c963f66afa6",
+        "4d0201db-36f5-428b-8d11-4f0a65e960cc"
+    ],
+    "ServiceIds": [
+        "e654f217-d1b5-46be-a820-e93ba568dfac"
+    ],
+    "RateGroupIds": [
+        "fe795f96-0b64-445b-89ed-c032563f2bac"
+    ],
+    "RateIds": [
+        "fe795f96-0b64-445b-89ed-c032563f2bac"
+    ],
+    "UpdatedUtc": {
```

Comment:
Perhaps we could add ValidFrom, ValidTo into request to filter out exceptions by seasonality.